populate/src/main/resources/seed/portals/demo/studies/heartdemo/enrollees/jsalk.json

```diff
@@ -79,7 +79,7 @@
   "kitRequestDtos": [
     {
       "creatingAdminUsername": "staff@heartdemo.org",
-      "kitTypeName": "saliva",
+      "kitTypeName": "SALIVA",
```

Comment:
Prior mis-configuration that will need to be updated in demo/prod.

Reply:
I think your migration script will handle that -- the kit type name isn't stored directly anywhere except in the kit type table, right?

Reply:
Correct. My comment here predates the addition of the migration script (fast-follow commit to handle Github Action failures), so I agree that this should be all set.